### PR TITLE
document both Scala 3 and 2 seeds, putting 3 first

### PR DESCRIPTION
**Pull request by SethTisue**

### CODE REVIEW

File: src/landing/_template/landing.st [hunk 1]
```diff
@@ -79,7 +79,8 @@ You just need one line of <code>build.sbt</code> to get started with Scala. Lear
 <h3>sbt new</h3>
 Choose from community-maintained <a href="https://github.com/search?o=desc&p=1&q=g8&s=stars&type=Repositories">Giter8 templates</a> to jump start your project:
 
-<pre><code class="prettyprint">\$ sbt new scala/scala-seed.g8
```
Comment (eed3si9n) [inline src/landing/_template/landing.st:82]: I suggest we copy paste `sbt init` output.

File: src/reference/00-Getting-Started/02-sbt-by-example.md [hunk 2]
```diff
@@ -541,13 +541,14 @@ or a continuous test like `~testQuick`.
 You can use the sbt `new` command to quickly setup a simple "Hello world" build.
 
 ```
-\$ sbt new scala/scala-seed.g8
-....
-A minimal Scala project.
+\$ sbt new scala/scala3.g8
```
Comment (eed3si9n) [inline src/reference/00-Getting-Started/02-sbt-by-example.md]: scala/toolkit.local also defaults to Scala 3:

```bash
Select a template: a
Scala version (default: 3.3.0):
Scala Toolkit version (default: 0.2.0):
```

File: src/landing/_template/landing.st (hunk 1 above)
Comment (SethTisue) [reply src/landing/_template/landing.st:82]: I've added a commit that does what I imagine you mean.

File: src/reference/00-Getting-Started/02-sbt-by-example.md (hunk 2 above)
Comment (SethTisue) [reply src/reference/00-Getting-Started/02-sbt-by-example.md]: Added a commit with the adjustment I think you're suggesting.

**Review by eed3si9n — APPROVED** [review]
Thanks!

**Pull request merged**